scripts/build-pre-release-test.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/sh -e
```

Comment:
Consider running shellcheck.

Reply:
I'm old-school Bourne shell.  Can you give me some hints as to how to make `BASH_SOURCE...` work, and what "shellcheck" is?

Reply:
Figured out the `DIR=...` line from your `scripts/graph.sh`, and now I see what "shellcheck" is.